app/entry.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+// webpack assets
+require('!!file?name=[name].[ext]!./html/index.html');
+require('./scss/base.scss');
+require('./view/signup/_signup.scss');
+require('./view/signin/_signin.scss');
+
+// npm modules
+const angular = require('angular');
+const ngRoute = require('angular-route');
+
+// angular modules
+angular.module('ramble', [ngRoute])
+.config(['$routeProvider', function($routeProvider){
```

Comment:
SInce yall have some many routes, its a good idea to pull out this config and put it in its own module that could be required in and and addle to the angular ramble module :)